modules/DMZ/utils/config_options.py

```diff
@@ -0,0 +1,27 @@
+class ConfigOptions(object):
+
+    #Initialize the class
+    def __init__(self):
+        self.training_file_name = None
+        self.test_file_name = None
+        self.separator_type = None
+        self.prediction_type = None
+        self.target_column = None
+        self.get_config_options()
+        self.validate_options()
+
+    #Get input from user
```

Comment:
@ZakeryFyke @ASAAR Is this how we want to get input from the user? I'm about to open a discussion issue on this with my proposed idea.